config/config.go

```diff
@@ -33,15 +36,31 @@ type IngressConfig struct {
 func Get() *IngressConfig {
 
 	options := viper.New()
-	options.SetDefault("MaxSize", 10*1024*1024)
-	options.SetDefault("Port", 3000)
-	options.SetDefault("StageBucket", "available")
-	options.SetDefault("Auth", true)
-	options.SetDefault("KafkaBrokers", []string{"kafka:29092"})
-	options.SetDefault("KafkaGroupID", "ingress")
+
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
+		cfg := clowder.LoadedConfig
+
+		sb := os.Getenv("INGRESS_STAGEBUCKET")
+		bucket, _ := clowder.ObjectBuckets[sb]
+
+		options.SetDefault("Port", cfg.WebPort)
+		options.SetDefault("KafkaBrokers", fmt.Sprintf("%s:%v", cfg.Kafka.Brokers[0].Hostname, *cfg.Kafka.Brokers[0].Port))
+		options.SetDefault("MinioEndpoint", fmt.Sprintf("%s:%d", cfg.ObjectStore.Hostname, cfg.ObjectStore.Port))
+		options.SetDefault("MinioAccessKey", *cfg.ObjectStore.AccessKey)
```

Comment:
I think you will need to get the acces key from the bucket subsection for app-interface mode.